src/server/src/test/java/io/cassandrareaper/resources/RepairRunResourceTest.java

```diff
@@ -364,7 +364,7 @@ public void testPauseNotRunningRun() throws InterruptedException, ReaperExceptio
   public void testPauseNotExistingRun() throws InterruptedException, ReaperException {
     RepairRunResource resource = new RepairRunResource(context);
     Response response = resource.modifyRunState(uriInfo, UUIDs.timeBased(),
-        Optional.of(RepairRun.RunState.PAUSED.toString()));
+        Optional.of(RepairRun.RunState.PAUSED.toString()), Optional.absent());
     assertEquals(Response.Status.NOT_FOUND.getStatusCode(), response.getStatus());
     assertEquals(0, context.storage.getRepairRunsWithState(RepairRun.RunState.RUNNING).size());
   }
```

Comment:
it would be great with some tests against `RepairManager.editRepairRun(..)`

Reply:
Yes. You right. Will do it.

Reply:
I did it. Check it out.

Reply:
sry, where is it?